components/director/pkg/graphql/schema.graphql

```diff
@@ -130,6 +130,7 @@ input ApplicationRegisterInput {
 	eventDefinitions: [EventDefinitionInput!]
 	documents: [DocumentInput!]
 	integrationSystemID: ID
+	providerDisplayName: String
```

Comment:
https://github.com/kyma-project/kyma/blob/513309fe099cc13c9fd3e656917ef2b456da2fbd/docs/application-connector/06-01-application.md

Is that field supposed to be optional?

Reply:
No, I'm going to fix it.